mod_reforged/hooks/config/strings.nut

```diff
@@ -2019,12 +2028,12 @@ foreach (vanillaDesc in vanillaDescriptions)
 	}),
 	RF_SwordmasterBladeDancer = ::UPD.getDescription({
 		Fluff = "Let's Dance!",
-		Requirement = "Sword"
+		Requirement = "Non-Fencing Sword"
 		Effects = [{
 			Type = ::UPD.EffectType.Passive,
 			Description = [
-				"Gain additional [Initiative|Concept.Initiative] equal to the armor ignore percentage of your equipped sword.",
-				"When using a non-fencing sword, the [Action Point|Concept.ActionPoints] costs of non-AOE skills is reduced by " + ::MSU.Text.colorPositive(1) + " and these skills build up " + ::MSU.Text.colorPositive("25%") + " less [Fatigue.|Concept.Fatigue]",
+				"Gain [Initiative|Concept.Initiative] equal to the armor ignore percentage of your equipped sword.",
+				"The [Action Point|Concept.ActionPoints] cost of non-AOE skills is reduced by " + ::MSU.Text.colorPositive(1) + " and these skills build up " + ::MSU.Text.colorPositive("25%") + " less [Fatigue.|Concept.Fatigue]",
```

Comment:
We currently don't restrict this entire perk to Fencing Swords. Only the AP cost part was restricted away from them. We can restrict the entire perk if you prefer this design but then this has to happen in a separate PR/commit from this strings improvement PR.

Reply:
non-restricted is fine if someone wants to bredit a mega swordsman or something